rye/src/cli/rye.rs

```diff
@@ -597,17 +628,12 @@ fn perform_install(
             echo!("For more information read https://mitsuhiko.github.io/rye/guide/installation");
         }
     }
+    // On Windows, we add the rye directory to the user's PATH unconditionally.
```

Comment:
I think at this point it might also sense to align the behavior.

Reply:
I can follow up with this in a separate PR. This way we separate the concerns a bit and can merge one but not the other if we feel like it.